packages/crayons-core/src/components/accordion/accordion.stories.mdx

```diff
@@ -36,11 +36,11 @@ import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
                 <fw-icon
                   name='minus'
                   size="14"
-                  slot="expanded-icon"></fw-icon>
+                  slot="expanded-icon" library="system" ></fw-icon>
```

Comment:
Cant you set it up so that if library is not mentioned , it defaults to using our default icon library ? Basically in your code you set library prop default value to "system" else all components have to be changed as in this PR which has resulted in 57 changed files!

Reply:
It does not seem possible as we provide crayons icons as the default set. System icon is a subset but we cannot set it to default. Also, there is no way we can make out if it's used from an internal component or an external component. In Shoelace also they are passing library='system' in all internal components.

Reply:
I tested all those components and its working. I tested by making it fail, then enabling the system mode. We can also do some quick sanity testing by individual developers who worked on that component. They can pull this branch and test it.